src/featurelistmodel.cpp

```diff
@@ -156,18 +167,67 @@ void FeatureListModel::reloadLayer()
 
   QgsFeature feature;
 
-  QStringList displayStrings;
-  QVariantList keys;
+  QList<Entry> entries;
+
+  if ( mAddNull )
+    entries.append( Entry( QStringLiteral( "<i>NULL</i>" ), QVariant( QVariant::Int ) ) );
```

Comment:
Where would this entry be after sorting? Probably it would be good to have it on the start or end...

Reply:
See above :)